src/components/sellInfo/index.js

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react';
+import "./style.scss";
+
+//const classname = require('classnames');
+
+class SellInfo extends Component {
+    constructor(props) {
+        super(props);
+        this.handleFromChange = this.handleFromChange.bind(this);
+        this.handleToChange = this.handleToChange.bind(this);
+        this.handleDateChange = this.handleDateChange.bind(this);
+        this.handleFlightNumberChange = this.handleFlightNumberChange.bind(this);
+    }
+
+    handleFromChange(e) {
+        this.props.onFromChange(e.target.value)
+    }
+
+    handleToChange(e) {
+        this.props.onToChange(e.target.value)
+    }
+
+    handleDateChange(e) {
+        this.props.onDateChange(e.target.value)
+    }
+
+    handleFlightNumberChange(e) {
+        this.props.onFlightNumberChange(e.target.value)
+    }
+
+    render() {
+        const { searchUsingFlightNumber, searchParams}= this.props;
```

Comment:
Should rename all search and searchParams to createListingFields or something like that